src/openforms/registrations/contrib/zgw_apis/tests/test_backend.py

```diff
@@ -1062,6 +1085,91 @@ def test_submission_with_zgw_backend_override_attachment_iotype(self, m):
             document_create_attachment2_body["informatieobjecttype"],
             "https://catalogi.nl/api/v1/informatieobjecttypen/1",
         )
+        self.assertEqual(
+            document_create_attachment2_body["bronorganisatie"],
+            "000000000",
+        )
+        self.assertEqual(
+            document_create_attachment2_body["vertrouwelijkheidaanduiding"],
+            "openbaar",
+        )
+        self.assertIsNone(document_create_attachment2_body["titel"])
```

Comment:
not valid according to the API spec https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/gemma-documentregistratiecomponent/1.2.0/src/openapi.yaml#tag/enkelvoudiginformatieobjecten/operation/enkelvoudiginformatieobject_create - this is not nullable and must be a string with length 1-200

This also means a client side validator is needed to prevent titles > 200 chars being entered